pkgs/development/libraries/glib/gdb_script.patch

```diff
@@ -0,0 +1,46 @@
+diff --git a/glib/meson.build b/glib/meson.build
+index b2dd569e1..b013991a1 100644
+--- a/glib/meson.build
++++ b/glib/meson.build
+@@ -513,8 +513,7 @@ configure_file(
+ )
+ 
+ install_data('glib_gdb.py',
+-  install_dir : glib_pkgdatadir / 'gdb',
+-  install_tag : 'devel',
++  install_dir : get_option('datadir') / 'glib-2.0' / 'gdb',
+ )
+ 
+ # This is needed to make gdb find glib_gdb.py
+@@ -541,7 +540,7 @@ configure_file(
+   input: 'libglib-gdb.py.in',
+   output: 'libglib-2.0.so.@0@-gdb.py'.format(library_version),
+   configuration: gdb_conf,
+-  install_dir: gdb_install_dir,
++  install_dir: glib_libdir,
+   install_tag: 'devel',
+   install: gdb_install,
+ )
+diff --git a/gobject/meson.build b/gobject/meson.build
+index 78b732bf3..cf1e6204f 100644
+--- a/gobject/meson.build
++++ b/gobject/meson.build
+@@ -176,8 +176,7 @@ gobject_query = executable('gobject-query', 'gobject-query.c',
+   dependencies : [libglib_dep, libgobject_dep])
+ 
+ install_data('gobject_gdb.py',
+-  install_dir : glib_pkgdatadir / 'gdb',
+-  install_tag : 'devel',
++  install_dir : get_option('datadir') / 'glib-2.0' / 'gdb',
```

Comment:
I already have `result-dev/share/glib-2.0/gdb/gobject_gdb.py` without this?

Reply:
you are right, my bad.